src/ServiceControl/Infrastructure/Settings/DataStoreType.cs

```diff
@@ -6,4 +6,11 @@ public enum DataStoreType
         RavenDb = 2,
         SqlDb = 3
     }
+
+    public static class DataStoreConfig
+    {
+        public static string InMemoryPersistence = "ServiceControl.Persistence.InMemory.InMemoryPersistenceConfiguration, ServiceControl.Persistence.InMemory";
```

Comment:
Should we change the name of the property to e.g. `InMemoryPersistenceTypeFullyQualifiedName`?

Reply:
I have renamed as suggested